tests/tc648/run.sh

```diff
@@ -0,0 +1,5 @@
+set -e
```

Comment:
set -eu ?

Reply:
I don't know scripting, i just copied a folder from another test
Should i remove both lines and put that instead?